src/test/java/com/wikia/webdriver/common/core/elemnt/Wait.java

```diff
@@ -46,6 +46,38 @@ public WebElement forElementPresent(By by) {
   }
 
   /**
+   * Checks if the element is clickable on browser
+   * <p/>
+   * * @param element The element to be checked
+   */
+  public WebElement forElementClickable(WebElement element) {
+    changeImplicitWait(0, TimeUnit.MILLISECONDS);
+    try {
+      element.getTagName();
+    } catch (WebDriverException e) {
+      PageObjectLogging.log("INIT ELEMENT", "PROBLEM WITH ELEMENT INIT", true);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "PROBLEM WITH ELEMENT INIT" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "INIT ELEMENT" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)